### ISSUE TITLE
Add flags to package:install command

### ISSUE THREAD

**Issue by Alroniks** [modmore/Gitify#83]
Just for fix idea. I am going to send PR.

`gitify package:install Ace --save` - as in composer and others, installed package and write line to config
and flag `--provider providerName` - to which provider added line (if exists), by default - modx.com

Examples:
`gitify package:install Ace --save`
`gitify package:install pdoTools --save --provider modstore.pro`

**Comment by Mark-H** [comment]
Yay :d 

Maybe we can hook into the download to see what provider was used to download the package, so that doesn't have to be specified manually?

**Comment by Alroniks** [comment]
@Mark-H Yes, it can be do automatically, but sometimes needed ability specify by hand (pdoTools exists on two repos, but versions can be different)

**Comment by Alroniks** [comment]
And after install package needed run `gitify extract` if you have namespaces in config, because gitify build -f remove all

**Comment by Alroniks** [comment]
@Mark-H Hi Mark, this issue labeled as ready. Is it works now? I probably missed some of the updates.

**Comment by Alroniks** [comment]
Nice :) I planned fix this issue soon.

**Comment by Mark-H** [comment]
That label was set by a project management tool as "ready to be developed"
